emails/apps.py

```diff
@@ -60,9 +60,6 @@ def _load_terms(self, filename: str) -> list[str]:
                 terms.append(word.strip())
         return terms
 
-    def ready(self):
-        import emails.signals  # noqa: F401 (imported but unused warning)
-
```

Comment:
(question, non-blocking) Any idea why we imported the signals during the ready? Wondering if we need to do this in the `privaterelay` app since the signals have been moved?